bitcoin/src/blockdata/fee_rate.rs

```diff
@@ -199,4 +225,20 @@ mod tests {
         let fee_rate = FeeRate(10).checked_div(0);
         assert!(fee_rate.is_none());
     }
+
+    #[test]
+    fn fee_convenience_functions_agree() {
```

Comment:
this test could be boilled down to like two lines:

```
fn fee_convenience_functions_agree() {
    let rate = FeeRate::from_sat_per_vb(1).expect("1 sat/byte is valid");
    assert_eq!(rate.fee_vb(1), rate.fee_wu(Weight::from_wu(4)));                           
}
```

after all, no need to test all of the transaction serialization stuff again here.